source/build/ssl_certificates.md

```diff
@@ -5,14 +5,18 @@
 ```
 We offer a range of certificates from Single Domain, Multiple Domain to Wildcard Certificates each ranging from Domain Control Validated to Extended Validated certificates.
 
+**Single-Domain SSLs**
```

Comment:
Two things for this..

1. is this a duplicate of https://docs.ukfast.co.uk/domains/ssl/types.html?
2. I would remove the tables, as they don't add or clarify anything - rather they seem to duplicate the bullet points.

Reply:
Potentially has the same content, however this is just for build docs so we need a section on SSLs without linking outside. Agreed, I've removed the tables.